packages/wethegit-components/src/components/in-view/preset-examples.stories.tsx

```diff
@@ -6,7 +7,7 @@ import { animation } from "./components"
 
 type Story = StoryObj<typeof InView>
 
-const Wrapper = ({ children }) => (
+const Wrapper: FC = ({ children }) => (
```

Comment:
@andrewrubin any particular reason for this?

Reply:
Updated this to accept a pre-defined type of ReactNode.